aten/src/ATen/templates/TypeDefault.cpp

```diff
@@ -53,7 +53,7 @@ void TypeDefault::set_data(Tensor & self, Tensor new_data) const {
 }
 
 Type & TypeDefault::toBackend(Backend b) const {
-  return at::globalContext().getNonVariableType(b,scalarType());
+  return at::globalContext().getNonVariableType(b,typeMetaToScalarType(typeMeta()));
```

Comment:
This change is orthogonal and not making progress (typeMeta() needs to go), but I'd like to just get rid of scalarType() for now.